docs/src/add-services/redis.md

```diff
@@ -4,16 +4,12 @@ weight: 11
 sidebarTitle: "Redis"
 ---
 
-Redis is a high-performance in-memory object store, well-suited for application level caching.
-
-See the [Redis documentation](https://redis.io/documentation) for more information.
-
+[Redis](https://redis.io/documentation) is a multi-model database that allows you to store data in memory
+for high-performance data retrieval and storage purposes.
 Platform.sh supports two different Redis configurations:
 
-- Persistent: useful for key-value application data
-- Ephemeral: in-memory only, useful for application caching
-
-Otherwise, they're identical.
+- [Ephemeral Redis](#ephemeral-redis): for application-level caching with no persistent storage
+- [Persistent Redis](#persistent-redis): for fast application-level key-value persistent storage
```

Comment:
:lady_beetle: I'm not sure if there's not too many nouns modifying storage. It might be a little hard to parse, so maybe making it a little longer would help?